compiler/rustc_hir/src/hir.rs

```diff
@@ -4996,7 +5012,7 @@ mod size_asserts {
     static_assert_size!(StmtKind<'_>, 16);
     static_assert_size!(TraitItem<'_>, 88);
     static_assert_size!(TraitItemKind<'_>, 48);
-    static_assert_size!(Ty<'_>, 48);
+    static_assert_size!(Ty<'_>, 56);
```

Comment:
Ah, this is unfortunate. `Ty` is a very common type, an extra 8 bytes just to fit a boolean?